ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -21,7 +21,11 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
   def index = AuthAction { implicit request =>
     val progressBarPercentage: Double = db.progressBarRatio() * 100 // expected to be between 0 and 100
     val progressBarPercentageRounded = progressBarPercentage - progressBarPercentage % 0.001
-    Ok(views.html.app("Recipeasy", progressBarPercentageRounded))
+    val usersCount = db.usersCount()
+    val curatedRecipesCount = db.countRecipesInGivenStatus(Curated) + db.countRecipesInGivenStatus(Verified) + db.countRecipesInGivenStatus(Finalised)
+    val finalisedRecipesCount = db.countRecipesInGivenStatus(Finalised)
```

Comment:
This is the same call as you're making above. Therefore you can assign the result to a val and use that in both cases removing the need to make 2 DB calls rather than 1.